tools/accuracy_checker/accuracy_checker/metrics/classification.py

```diff
@@ -353,3 +350,35 @@ def evaluate(self, annotations, predictions):
     def reset(self):
         self.targets = []
         self.results = []
+
+class AcerScore(PerImageEvaluationMetric):
+    __provider__ = 'acer_score'
+    annotation_types = (ClassificationAnnotation, TextClassificationAnnotation)
+    prediction_types = (ClassificationPrediction, )
+
+    def configure(self):
+        if isinstance(confusion_matrix, UnsupportedPackage):
+            confusion_matrix.raise_error(self.__provider__)
+        self.reset()
+
+    def update(self, annotation, prediction):
+        self.targets.append(annotation.label)
+        self.results.append(prediction.label)
+        return 0
```

Comment:
please return some more meaningful value if it is possible.
metric result for each iteration used in quantization scope, so now your model can not be quantized by Accuracy Aware algo

Reply:
at least accuracy for image (annotation.label == prediction.label)